lib/iruby/ostream.rb

```diff
@@ -25,9 +25,10 @@ def read(*args)
     alias_method :next, :read
     alias_method :readline, :read
 
-    def write(s)
+    def write(*args)
+      str = args.join
```

Comment:
I think the following code is better than using `Array#join` for mimicking `IO#write`'s behavior.

```
str = StringIO.open {|sio| sio.write(*args); sio.string }
```

Reply:
@kojix2 Could you please rewrite also puts and printf by using StringIO?

Reply:
I'll try.

Reply:
What should I do next?